docs/run-instance/running-with-docker.md

```diff
@@ -1,247 +1,509 @@
+        └── ors.log
+
+8 directories, 35 files
+```
+
+* `example-ors-config.env`: env file template. A customized copy of this file can be used in combination with the `docker-compose.yml`, see [Set properties in an environment file](#set-openrouteservice-properties-in-an-environment-file). Do not edit this file directly since it will be overridden by docker! 
+* `example-ors-config.yml`: YAML file template. A customized copy of this file can be used as `ors-config.yml`, see [Use customized config file](#use-customized-config-file). Do not edit this file directly since it can be overridden by docker!
+* `ors-config.yml`: The default config used by the openrouteservice inside the container. This file will not be overridden by docker. Initially, the file is identical to `example-ors-config.yml`. But if a newer docker image tag is used in the `docker-compose.yml`, the `example-ors-config.yml` might be updated if in the newer version of openrouteservice there are changes in the configuration properties or their default values. The updated file can be compared with the customized `ors-config.yml`. See [Use customized config file](#use-customized-config-file)   
+* `elevation_cache`: Directory, where openrouteservice stores downloaded elevation data to avoid repeated downloads.
+* `example-heidelberg.osm.gz`: The initial sample OSM data used to run the container in its initial setup. Normally, you will remove this file or add another OSM file to the same directory and adapt your config to use your file.
+* `graphs`: Directory for the generated graphs. Initially, there is only `car` because this is the only enabled routing profile in the default `ors-config.yml`
+* `logs`: Here, the openrouteservice logs are stored. :warning: Logs of several runs will be accumulated here. In contrast, the logs you get with `docker compose logs` are only logs of the current run, but on the other hand they contain very helpful logging from docker or the entrypoint script, which is not visible in the files in `logs`.    
+
+
+### Avoid files owned by root
+
+With the default `docker-compose.yml`, the generated files are owned by root. 
+If you don't have root permissions (sudo) you e.g. cannot edit or add files in `ors-docker/config`. 
+Of course, you can configure openrouteservice with the help of the `docker-compose.yml` (see [here](#set-openrouteservice-properties-in-docker-composeyml)) or a referenced environment file (see [there](#set-openrouteservice-properties-in-an-environment-file)), 
+which needs not to be located in the config folder because the openrouteservice running inside docker don't need to 'see' the environment file. 
+
+But if you for example want to save a new pbf file to `ors-docker/files`, then you need sudo rights.
+
+To avoid root ownership of the generated folders and files, 
+
+1. uncomment the 'user' line in your `docker-compose.yml`:
+    ```yaml
+        #user: "1000:1000" // [!code --]
+        user: "1000:1000" // [!code ++]
+    ```
```

Comment:
This is only true, if the host user UID and GID are 1000:1000 as well. Else it'll clash again, and the user will need sudo. Chicken-and-egg problem.

I'd say we should just describe three scenarios here:
1. Run as is -> Sudo
2. Run as 1000:1000 user. Implies the host user is either part of group 1000 and/or user with id 1000 or has the needed permissions to chown files and folders.
3. Run with your host UID and GID -> Requires to build the docker image from source with the IDs as build-args.

Reply:
Explained below

Reply:
fair enough.